crates/gdtk-parser/src/error.rs

```diff
@@ -1,5 +1,3 @@
-use gdtk_diag::{Diagnostic, Span};
-
 #[derive(Debug, thiserror::Error)]
```

Comment:
should probably remove this file altogether. it'll be refactored in future anyway